Makefile

```diff
@@ -366,6 +367,16 @@ gotest: $(SOURCE_GENERATED) ## Trigger minikube test
 	$(if $(quiet),@echo "  TEST     $@")
 	$(Q)go test -tags "$(MINIKUBE_BUILD_TAGS)" -ldflags="$(MINIKUBE_LDFLAGS)" $(MINIKUBE_TEST_FILES)
 
+out/test-report.json: $(SOURCE_FILES) $(GOTEST_FILES)
+	$(if $(quiet),@echo "  TEST     $@")
+	$(Q)go test -tags "$(MINIKUBE_BUILD_TAGS)" -ldflags="$(MINIKUBE_LDFLAGS)" $(MINIKUBE_TEST_FILES) \
+	-json -coverprofile=out/coverage.out > out/test-report.json
+
+out/coverage.out: out/test-report.json
+coverage.html: out/coverage.out
```

Comment:
would u plz add comments, I worry this get confused with `make html-report` that converst last integraiton test run to gopogh

Reply:
I don't think it is possible to add help for non-phony targets, but I suppose I could make a plain old comment